### ISSUE TITLE
Marvin Windows error: pwd module not found

### ISSUE THREAD

**Issue by sjshamsi** [sdss/marvin#429]
I did a new Marvin install via pip on a Windows 10 machine, and it seemed to have installed correctly, but it runs into an error while importing Marvin:
```import marvin

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-1-86893d82dac6> in <module>()
----> 1 import marvin

C:\Users\sshamsi\AppData\Local\Continuum\anaconda3\lib\site-packages\marvin\__init__.py in <module>()
     26 from marvin import extern
     27 
---> 28 from marvin.core.exceptions import MarvinUserWarning, MarvinError
     29 from brain.utils.general.general import getDbMachine
     30 from brain import bconfig

C:\Users\sshamsi\AppData\Local\Continuum\anaconda3\lib\site-packages\marvin\core\exceptions.py in <module>()
     19 import sys
     20 import warnings
---> 21 import pwd
     22 
     23 import marvin

ModuleNotFoundError: No module named 'pwd'
```
**Desktop:**
 - OS: Windows 10
 - Jupyter Notebooks on Chrome (installed via Anaconda)
 - Marvin 2.2.5

**Comment by albireox** [comment]
Hi @sjshamsi,

The `pwd` module is only available for Unix systems as explained [https://docs.python.org/2/library/pwd.html](here). It's very useful that you tried this and reported it, but supporting Marvin on Windows machines is at the time out of our scope. 

I have created a milestone for Windows support and added you issue there, but I don't think we have a timeframe to have this fixed. In the meantime I recommend you install a Linux virtual machine and run Marvin from there.

**Comment by havok2063** [comment]
related to #397

**Comment by havok2063** [comment]
This has been resolved with PR #652 and merged in.  It will be available in the next release of Marvin.
